waiter/src/waiter/request_log.clj

```diff
@@ -93,7 +93,11 @@
 (defn log-request!
   "Log a request"
   [request response]
-  (log (merge (request->context request) (response->context response))))
+  (let [redacted-request-fields-string (get-in response [:descriptor :service-description "env" "REDACTED_REQUEST_FIELDS"])]
```

Comment:
I feel like we should be prefixing our "reserved" environment variable names:
`WAITER_REDACTED_REQUEST_FIELDS`?

Reply:
By "reserved" I mean "we reserve the right to take some action if you set this."

`WAITER_CONFIG_`? We'd have to exclude this from the regex. @shamsimam ?

Reply:
Sounds good to me. 
We should also add checks in our code that disallows waiter code from generating environment variables starting with `WAITER_CONFIG_`.